queries.sql

```diff
@@ -8,3 +8,55 @@ SELECT name,escape_attempts FROM animals WHERE  weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals  WHERE weight_kg >=10.4 AND  weight_kg <=17.3;
+
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name ILIKE '%mon%';
+
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL ;
+
+COMMIT;
+
```

Comment:
- [ ] Kindly write this transaction based on the requirements for this milestone:

- _**Inside a transaction update the `animals` table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.**_

#### Please attach the screenshots of these transactions in the pull request. 👍 💻